Revit_oM/Generic/TypeMap.cs

```diff
@@ -20,20 +20,24 @@
  * along with this code. If not, see <https://www.gnu.org/licenses/lgpl-3.0.html>.      
  */
 
+using BH.oM.Base;
 using System;
 using System.Collections.Generic;
-
-using BH.oM.Base;
+using System.ComponentModel;
 
 namespace BH.oM.Adapters.Revit.Generic
 {
+    [Description("An entity defining the relationship between property names of BHoM type and parameter names of correspondent Revit elements.")]
     public class TypeMap : BHoMObject
     {
         /***************************************************/
         /****             Public Properties             ****/
         /***************************************************/
 
+        [Description("BHoM type being, which property names are being mapped with Revit element parameters.")]
```

Comment:
Would check the wording of this. "BHoM type being, which property..." reads a bit weird imo. Do we need the word 'being'?

Reply:
Good catch - it was a legacy after some previous thought. Deleted.